helm_chart/ibm-storage-enabler-for-containers/cv-tests/test-default/values.yaml

```diff
@@ -0,0 +1,169 @@
+# ---------------------------------------------------------
+# Helm chart to install IBM Storage Enabler for Containers.
+# Enables IBM Storage with Kubernetes by implementing Kubernetes Dynamic Provisioner and FlexVolume.
+#
+# IBM Storage Enabler for Containers includes the following main images:
+#   - deployment/ubiquity                 : A mediator between IBM Storage and k8s FlexVolume \ Dynamic Provisioner.
+#   - deployment/ubiquity-db              : Stores meta-data for the dynamic provisioned volumes.
+#   - deamonset/ubiquity-k8s-flex         : Implements k8s FlexVolume driver.
+#   - deployment/ubiquity-k8s-provisioner : Implements k8s Dynamic Provisioner
+# ---------------------------------------------------------
+# Operation backend of Provisioner and Flex volume.
+# IBM Storage Enabler for Containers supports one of the following backend types: spectrumConnect OR spectrumScale.
+backend: spectrumConnect
+
+
+# IBM Storage Enabler for Containers supports one of the following backend types: spectrumConnect OR spectrumScale.
+# Select a backend that you intend to use and comment out the other backend section.
+spectrumConnect:
+  connectionInfo:
+    # IP\FQDN and port of Spectrum Connect server.
+    fqdn: 9.37.37.46
```

Comment:
what is this IP?  its related to icp testing in ci?

Reply:
Yes, A SC on ICP CI env